### ISSUE TITLE
GBA filter?

### ISSUE THREAD

**Issue by suloku** [Cruel/freeShop#217]
I know there's only 10 gba ambassador games on the official shop and that probably there will never be any more, but I think it would be worth that they get their own filter in the platform section.

As of now, none of the current platform filters will display those games, so the only way to find them is either without any filter or deliverately searching the game's, and I think it would be great to have a Game Boy Advance filter in the list, even if it will only ever display those 10 games.

Also, what about an icon to display the game's platform? (DSIWare, VCNes, VCSNES....) I think it would improve navigation when searching franchises that have games in different platforms, both next to the region flag and in the bottom screen.

And thanks for this great app, first time I used it, and is awesomely well made and polished.

**Comment by Multipixelone** [comment]
I agree with both the GBA filter, and a icon, I think they would both be welcome additions.
